### PR TITLE
Add new Coaster Type Maurer

### PR DESCRIPTION
**Pull request by Realsteel89**
**Adding the new Coaster Type Maurer**
Maurer
The Track is Based on the Maurer Spinning Coaster in Real Life and based as the MiniRollerCoaster from RCT2

**What I did:**
- Building the new Track Model and render most of the sprites there are possible to the Coaster Track
- limited it with the cheat function that only the track parts are possible without cheats that the real coaster has too
- changing Supports from fork to tubes
- changing some hights from supports to look nicer
- adding vertical lift animation sprites to the coaster too (manually edding)
grafik

- adding an preview Image to coaster Type
- changing the rendering sprites from small vertical loops because had some cutes by the border of the tiles so i edding it   manually to look correct

**Mistakes that i need to fix now:**

- Fixing the small loop sprites and bounding boxes (they currently glitch badly, although removing the small loops entirely is also an option)
- fix maybe some hights of support

_this track type can be so useful for more Coaster Types like_ 
**Gerstlauer Eurofighter**
grafik
**Gerstlauer Spinner**
grafik
**Vekoma Family Coaster**
grafik

and so many one more....

It has 1274 Sprites so most of the Elements that the game can do at the moment limited by cheat setting on/off

_**later plans:**_
-maybe realistic spinning maurer car for the track
- vertical tower lift and an 110° Drop sprite
grafik
- maybe build animated breaks like the Minicoaster has with open/close animation

**Important:** **The Problem that it exist actually no Car for the Track, you need to choose it via cheat funktion!**

I would appreciate if someone could test to ensure that the branch is functional and I didn't forget to commit any sprites.

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -220,6 +220,12 @@ The following people are not part of the development team, but have been contrib
 * Joel H. (HtotheTML)
 * John Mulcahy (jayjay300)
 * Chase Percy (Chase-Percy)
+<<<<<<< maurer
+=======
+* Wenzhao Qiu (qwzhaox)
+* Michael Bernardi (mrmbernardi)
+* Aram Kazorian (aramk-hub)
+>>>>>>> develop
```
Comment (AaronVanGeffen) [inline contributors.md]: This looks like an unresolved merge conflict.

Comment (Realsteel89) [reply contributors.md]: I think i fixed these conflict now

File: contributors.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ The following people are not part of the development team, but have been contrib
 * Umar Ahmed (umar-ahmed) - MacOS file watcher
 * Andrew Arnold (fidwell) - Added window support for more scenery groups.
 * Josh Trzebiatowski (trzejos) - Ride and scenery filtering
-* (kyphii) - Extended color selection, reversed ride vehicles, misc.
+* (kyphii) - Extended color selection
```
Comment (fidwell) [inline contributors.md:107]: Looks like an accidental revert.

```suggestion
* (kyphii) - Extended color selection, reversed ride vehicles, misc.
```

File: CMakeLists.txt [hunk 3]
```diff
@@ -68,9 +68,9 @@ set(OBJECTS_VERSION "1.3.11")
 set(OBJECTS_URL  "https://github.com/OpenRCT2/objects/releases/download/v${OBJECTS_VERSION}/objects.zip")
 set(OBJECTS_SHA1 "02ddf6c685a2da8bac98a57b34be95d63192443d")
 
-set(OPENSFX_VERSION "1.0.3")
+set(OPENSFX_VERSION "1.0.2")
```
Comment (fidwell) [inline CMakeLists.txt:71]: Looks like an accidental revert.

```suggestion
set(OPENSFX_VERSION "1.0.3")
```

File: CMakeLists.txt [hunk 4]
```diff
@@ -68,9 +68,9 @@ set(OBJECTS_VERSION "1.3.11")
...
 set(OPENSFX_URL  "https://github.com/OpenRCT2/OpenSoundEffects/releases/download/v${OPENSFX_VERSION}/opensound.zip")
-set(OPENSFX_SHA1 "ffec5a97bd5035860c4c9a43fe32cf51886478e3")
+set(OPENSFX_SHA1 "64EF7E0B7785602C91AEC66F005C035B05A2133B")
```
Comment (fidwell) [inline CMakeLists.txt:73]: Looks like an accidental revert.

```suggestion
set(OPENSFX_SHA1 "ffec5a97bd5035860c4c9a43fe32cf51886478e3")
```

File: src/openrct2/ride/coaster/meta/GigaCoaster.h [hunk 5]
```diff
@@ -54,7 +54,7 @@ constexpr RideTypeDescriptor GigaCoasterRTD =
     )),
     SET_FIELD(ColourPreview, { SPR_RIDE_DESIGN_PREVIEW_GIGA_COASTER_TRACK, SPR_RIDE_DESIGN_PREVIEW_GIGA_COASTER_SUPPORTS }),
     SET_FIELD(ColourKey, RideColourKey::Ride),
-    SET_FIELD(Name, "giga_rc"),
+    SET_FIELD(Name, "GIGA_RC"),
```
Comment (fidwell) [inline src/openrct2/ride/coaster/meta/GigaCoaster.h:57]: Looks like an accidental revert.

```suggestion
    SET_FIELD(Name, "giga_rc"),
```

File: contributors.md (hunk 2 above)
Comment (duncanspumpkin) [reply contributors.md:107]: Yeah there is a lot of reverts in this pr.

**Comment by duncanspumpkin** [comment]
I've converted this to draft whilst it's still got all these merge issues. Please rebase and clean up the commit history so that it doesn't revert anyone else's work and is purely your change.

File: src/openrct2/ride/coaster/meta/GigaCoaster.h (hunk 5 above)
Comment (Realsteel89) [reply src/openrct2/ride/coaster/meta/GigaCoaster.h:57]: yes i made a mistake back then i had no stats left on my coaster and had deleted the gigabahn for whatever reason. i think i wanted to replace something i clicked replace all but not only in the current form but in the whole project.

**Comment by Realsteel89** [comment]
> I've converted this to draft whilst it's still got all these merge issues. Please rebase and clean up the commit history so that it doesn't revert anyone else's work and is purely your change.

I will do this because its an long branch i used and i tested so mutch makes fals and its so nessesarry for the pr that i need to delete the useless comits for it

**Comment by Realsteel89** [comment]
I will rebuild the branch again I have so many changes that have nothing to do with the track. I have now brought my develop to current status I think this is more useful than this sausage salad to submit as PR.

**Pull request closed without merging**